src/components/SecondaryButton/SecondaryButton.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SecondaryButtonWrapper } from './SecondaryButton.styles';
+
+const SecondaryButton = ({ content, type = 'button', onClick }) => {
+  return (
+    <SecondaryButtonWrapper type={type} onClick={onClick}>
+      {content}
+    </SecondaryButtonWrapper>
+  );
+};
+
+SecondaryButton.propTypes = {
+  content: PropTypes.string.isRequired,
+  type: PropTypes.oneOf(['button', 'submit', 'reset']),
+  onClick: PropTypes.func
```

Comment:
Just got to thinking, should we require the `onClick` prop? Not necessary but would avoid accidentally having buttons that don't do anything. I didn't do this on PrimaryButton but if you think it's worthwhile I'll update it.

What do you think?

Reply:
No, you don't necessarily need one. In forms for example, the event is handled my the submission of the form as opposed to the button so a onClick is not always required.

Reply:
Thanks for taking a look Shane, and for flagging up whether we require onClick - and thanks for the clarity on that Rob. Makes sense to me! I'll go ahead and squash and merge.